app/models/validators/asylum_support_decision_validator.rb

```diff
@@ -14,6 +15,16 @@ class AsylumSupportDecisionValidator < SimpleDelegator
   validates :tribunal_decision_judges, presence: true
   validates :tribunal_decision_landmark, presence: true
   validates :tribunal_decision_reference_number, presence: true
-  validates :tribunal_decision_sub_category, presence: true
+  validates :tribunal_decision_sub_category, tribunal_decision_sub_category: true
```

Comment:
`tribunal_decision_sub_category_relates_to_parent` was only a suggestion. The class should be named for how it's being validated. `validates :thing, date: true` is pretty self explanatory. If it's validating that it's related to a parent then `tribunal_decision_sub_category_relates_to_parent` is good. If it's doing more than that, maybe split the logic out into separate validators?